magentov2/plugin/Indodana/PayLater/registration.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+\Magento\Framework\Component\ComponentRegistrar::register(
```

Comment:
What is this code for? Can you make code comment?

Reply:
Register the module into magento system